specs/pvs/corla.pvs

```diff
@@ -922,8 +995,10 @@ END cast_vote_records
 %%% the county-level, and one is for audit board members.
```

Comment:
@aisamanra? Do you mean @ranweiler or Morgan? To the best of my knowledge, Getty Ritter is not working on this project.